src/Components/HomePage.js

```diff
@@ -49,7 +57,7 @@ const styles = StyleSheet.create({
     backgroundColor: theme.BACKGROUND_COLOR_LIGHT,
   },
   menuItem: {
-    height: '30%',
+    height: '20%',
```

Comment:
25% maybe ? i have a quite weird white banner at the bottom of my homepage now

Reply:
I weent back to 30% since the home page does not change in the end :)